src/http/response.ts

```diff
@@ -90,10 +90,18 @@ export class Response {
     this.headers = new Headers();
     this.template_engine = options.template_engine;
     this.views_path = options.views_path;
-    this.headers.set(
-      "Content-Type",
-      this.getContentTypeFromRequestAcceptHeader(),
-    );
+    if (this.options && this.options.default_response_content_type) {
+      //const defaultContentType = this.options.default_response_content_type ?? "application/json";
```

Comment:
is this needed?

Reply:
The comment? No i think i left that inn by mistake :X Remove it now though